app/test_main.py

```diff
@@ -1,3 +1,29 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_human_age_zero() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_human_age_in_1_year() -> None:
+    assert get_human_age(15, 15) == [1, 1]
+
+
+def test_ages_between_15_and_24() -> None:
+    assert get_human_age(22, 22) == [1, 1]
+
+
+def test_human_age_in_2_year() -> None:
+    assert get_human_age(24, 24) == [2, 2]
+
+
+def test_difference_between_cat_dog_year() -> None:
+    assert get_human_age(28, 28) == [3, 2]
+
+
+def test_human_age_in_3_year() -> None:
+    assert get_human_age(28, 29) == [3, 3]
+
+
+def test_large_ages() -> None:
+    assert get_human_age(100, 100) == [21, 17]
```

Comment:
Consider using @pytest.mark.parametrize to test multiple scenarios with the same test function. This will make your tests cleaner and more maintainable, as suggested by the checklist.